src/EnlistmentInfo.targets

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
...unless cake can be downloaded as a package as a pre-build step with no elevated install required. In which case, I'm much more open to it.